packages/cli/src/commands/dev/dev.ts

```diff
@@ -93,18 +93,22 @@ export default async function dev(
   const timeout = setTimeout(async () => {
     if (link.status !== 'linked') return;
 
-    let refreshCount = 0;
-    for await (const token of refreshOidcToken(
-      controller.signal,
-      client,
-      link.project.id,
-      envValues,
-      'vercel-cli:dev'
-    )) {
-      output.debug(`Refreshing ${chalk.green(VERCEL_OIDC_TOKEN)}`);
-      envValues[VERCEL_OIDC_TOKEN] = token;
-      await devServer.runDevCommand(true);
-      telemetry.trackOidcTokenRefresh(++refreshCount);
+    try {
```

Comment:
Required to ensure an error from the dev server is surfaced to the user and not an `AbortError`